docs/src/examples/solvers/Lazy-Ellipsoids-Abstraction.jl

```diff
@@ -26,9 +26,7 @@ concrete_problem = NonLinear.problem()
 concrete_system = concrete_problem.system
 
 # Optimizer's parameters
-const FALLBACK_URL = "mosek://solve.mosek.com:30080"
-sdp_opt = optimizer_with_attributes(Mosek.Optimizer, MOI.Silent() => true)
-MOI.set(sdp_opt, MOI.RawOptimizerAttribute("fallback"), FALLBACK_URL)
+sdp_opt = optimizer_with_attributes(COSMO.Optimizer, "verbose" => false)
```

Comment:
```suggestion
sdp_opt = optimizer_with_attributes(COSMO.Optimizer, MOI.Silent() => true)
```

Reply:
this should work for any solver. It's better as then you don't need to remember the name of the option for each solver